src/main/java/frc/robot/subsystems/BallPath/Shooter/PIDShooterImpl.java

```diff
@@ -68,6 +97,72 @@ public void setShotPosition(ShotPosition shotPosition) {
         this.requestedPosition = shotPosition;
     }
 
+    public double getSetpointHood(double distance){
+        double hoodDif, distDif, difFromUpper, percentToAdd, amountToAdd;
+        double returnAmount = 0;
+        Integer Fender = 100_000;
+        Integer Tarmac = 230_000;
+        Integer CloseLaunchPad = 300_000; // pls test
+        Integer FarLaunchPad = 330_000; // pls test
+        Integer HumanPlayer = 400_000; // pls test
+        double[] Distances = {55.0, 153.0, 202.95, 244.77, 305.66};
+        Integer[] HoodValues = {100_000, 230_000, 300_000, 330_000, 400_000};
```

Comment:
Typical Java code convention, and what we've been using elsewhere in this project, is that variables are named in `lowerCamelCase`, not `PascalCase`. So this line for example should look like `int[] hoodValues = { ... }`.